api/src/pcapi/core/offers/factories.py

```diff
@@ -131,6 +131,7 @@ class Meta:
     motorDisabilityCompliant = False
     visualDisabilityCompliant = False
     isEducational = False
+    subcategoryId = subcategories.SUPPORT_PHYSIQUE_FILM
```

Comment:
Il faut un identifiant ici, pas une sous-catégorie : 

```suggestion
    subcategoryId = subcategories.SUPPORT_PHYSIQUE_FILM.id
```